core/kamon-core-bench/src/main/scala/kamon/bench/ThreadLocalStorageBenchmark.scala

```diff
@@ -54,28 +54,3 @@ class ThreadLocalStorageBenchmark {
     FTLS.current()
   }
 }
-
-
-class OldThreadLocal extends Storage {
```

Comment:
@jatcwang please don't remove the `OldThreadLocal`(is our baseline in the microbenchmark), maybe is a good idea change the name to something like `BaselineThreadLocal` and in the test add `CrossThreadLocal` in order to measure against the baseline WDYT?